dvc/tree/base.py

```diff
@@ -242,7 +243,7 @@ def get_hash(self, path_info, **kwargs):
         )
 
         if not self.exists(path_info):
-            return self.PARAM_CHECKSUM, None
+            return None
```

Comment:
Can we just do `return HashInfo(self.PARAM_CHECKSUM, None)` here? Suggesting this because otherwise there will be 3 kinds of possible results here:
1. None
2. Regular HashInfo
3. HashInfo with no value.